tsconfig.json

```diff
@@ -10,13 +10,13 @@
     "resolveJsonModule": true,
     "downlevelIteration": true
   },
-  "exclude": ["./examples/"],
-  "references": [
-    {
-      "path": "./packages/sdk-client"
-    },
-    {
-      "path": "./packages/sdk-client-v3"
-    }
-  ]
+  "exclude": ["./examples/"]
+  // "references": [
+  //   {
+  //     "path": "./packages/sdk-client"
+  //   },
+  //   {
+  //     "path": "./packages/sdk-client-v3"
+  //   }
+  // ]
```

Comment:
```suggestion
```

better remove them as comments are not valid json

Reply:
They are valid in a `tsconfig.json` files 😀